_data/sidebar_tree.yaml

```
@@ -70,6 +70,11 @@ tree:
     tree:
     - url: /rules/archive
       title: Game Rules Archive
+  - url: /simulator
```

This url needs to match exactly in order for the showing of the current page to work. I think therefore it needs to end with a `/`:
```suggestion
  - url: /simulator/
```